.circleci/config.yml

```
@@ -41,7 +41,7 @@ executors:
       - image: ubuntu:24.04
   golangci-lint:
     docker:
-      - image: golangci/golangci-lint:v1.60.1
+      - image: golangci/golangci-lint:v1.64.5
```

Should the patch version be pinned here? Might be better to always pull in the latest patch for a given major/minor?
```suggestion
      - image: golangci/golangci-lint:v1.64
```